spec/helpers/results_helpers_spec.js

```diff
@@ -51,7 +51,8 @@ describe('ResultsHelpers', () => {
     });
 
     describe('pretty statement results when requested', () => {
-      it('for CMS107v6 correctly', () => {
+      xit('for CMS107v6 correctly', () => {
+        // TODO: Find another measure to use. PrincipalDiagnosis is no longer a QDM attribute and is used in this measure logic
```

Comment:
I don't think this test necessarily has to test the pretty result for principal diagnosis--could we just swap out the clause with another one on the same measure?

Reply:
On the other hand, since the tests below do need to be updated anyway, I could see doing them all in one batch.  Is there a future task to unskip these tests? Does that need to be done before any other significant work is done on cqm-execution? @jbradl11